### ISSUE TITLE
Exit with non-zero status if a one of the command fails

### ISSUE THREAD

**Issue by aswinkarthik** [karuppiah7890/snail-runner#1]
## Bug

Whenever a command fails, it does not exit with non-zero exit status

## Expected behaviour

`snail-runner` should exit with non-zero exit status

## Example

```
$ snail-runner "randomcommand"
The complete output of all the commands will be shown after running them all

Started running:  randomcommand

Output of the commands:

Result of running the below command:
$ randomcommand

It failed with exit code 127

Stderr:
/bin/sh: randomcommand: command not found


$ echo $?
0
```

**Comment by karuppiah7890** [comment]
hmm. considering that `snail-runner` only runs commands and it should give the exit status for it's own errors and the errors of the commands that it runs, I will try to implement both of these 😄 as both are not present currently

**Comment by karuppiah7890** [comment]
@aswinkarthik This is fixed in version [0.1.1](https://github.com/karuppiah7890/snail-runner/releases/tag/0.1.1)
